### ISSUE TITLE
Investigate changing non-url a elements to button

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#766]
Look into how hard it would be to change the `url` lacking `a` elements that mb2 uses into `button` elements.

Now that `cargo +nightly watch -x 'check --target=wasm32-unknown-unknown'` has pointed it out, I followed their link and see that using `a` for a link that doesn't have a url is poor form and, at least in theory, should be done by a `button`. I understand and basically agree, although I think mb2 uses `a` based on UIkit examples, so switching the `a` into `button` may be tricky. It also could be drop-dead simple, so this GitHub issue starts with a request that I look into how hard it will be. If its trivial, go ahead and fix it on this issue. If not, create a new issue with the details and close this one.

**Comment by ctm** [comment]
FWIW, I was thinking about this when I added the down-arrow to pop-out chat and made a CSS mod that allowed me to use a `button` rather than an `a`. However, it looks like my change broke the GitHub login button (#768), so I'll try to fix both of those things now.

**Comment by ctm** [comment]
I've fixed all the ones where i'm using a `uk-icon` attribute. I don't think I can trivially change the others, but I didn't think to try before starting the deploy for this evenings game.

So, I'll futz around a little more and see if any/all of the nine remaining offenses can be corrected.

**Comment by ctm** [comment]
It's not worth the hassle trying to fix the nine remaining instances. We're doing what UIkit suggests doing. If I simply change the `a` into `button` it looks pretty bad and would require a bunch of hacks to make it look the way it looks with the `a`.
